tests/manage/monitoring/prometheusmetrics/test_monitoring_defaults.py

```diff
@@ -97,6 +99,9 @@ def test_ceph_rbd_metrics_available():
     Ceph RBD metrics should be provided via OCP Prometheus as well.
     See also: https://ceph.com/rbd/new-in-nautilus-rbd-performance-monitoring/
     """
+    if storagecluster_independent_check() and skipif_ocs_version('<4.6'):
```

Comment:
Probably better to check the version config value as follows:

```python
if storagecluster_independent_check() and float(config.ENV_DATA['ocs_version']) < 4.6:
```

Reply:
Done